### DISCUSSION TITLE
Model management & Node Parameter management

### DISCUSSION THREAD

**Discussion by Craigjw** [Comfy-Org/ComfyUI#5748] [General]
Both these issues originate from a lack of ability to find the right files.

Model Management issue
When I download a workflow, there are a bunch of nodes, mostly for models, I know there is a model management page, but this does not help much when I want to find the model used by  the workflow creator, please allow some link functionality to allow a github link or something with the model.  It's a slog to try and find the model that it was originally intended to be used with.
There should be a link to the model indexed in the model manager.

Node parameters
This is abismal, some parameters require specific files, which sometimes don't have any descriptions what they are or where to get the file from.
An example of this can be found with the Comfyui fooocus inpait - load fooocus inpaint.
What the hell is the head and path, these are god aweful to find. 
If there are seperate files needed, there should be links to where these can be found with the manager some how.

**Comment by OldmanY** [comment]
I found that discussing plugins and nodes here is not very popular. Maybe it is some kind of taboo?

**Reply by ltdrdata** [reply]
Discussing custom nodes itself is not a taboo.

It's just that when issues arise with custom nodes, the creator of the custom node is likely to be the best person to address them. (Issues that arise within custom nodes are often something only the creator of the custom node can fix.)

Since it's physically impossible for the core team to track and respond to issues arising from thousands of custom nodes, we guide users to the appropriate place for resolution.

BTW, we have a forum for ComfyUI community.
https://forum.comfy.org/

**Comment by bigcat88** [comment]
> please allow some link functionality to allow a github link or something with the model.

this has been discussed several times in this repository, no one has suggested anything better than hashes yet.

I created a [discussion](https://github.com/Comfy-Org/ComfyUI_frontend/discussions/1806) in the front-end repository, because this should mostly be done in the front-end, so far we are waiting for the discussions to start :)
